debugger/src/debug_adapter/dap/mod.rs

```diff
@@ -0,0 +1,15 @@
+/// Implements the logic for each of the MS DAP types (events, requests, etc.)
```

Comment:
I think generally we prefer a `dap.rs` at the same level as the `dap/` dir instead of `mod.rs` files nowadays :)

Reply:
Good to know. I've updated other `mod.rs` in the debugger crate in the same way.